modules/virt-enabling-usb-host-passthrough.adoc

```diff
@@ -0,0 +1,67 @@
+// Module included in the following assemblies:
+//
+// * virt/advanced_vm_management/virt-configuring-usb-host-passthrough.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="virt-enabling-usb-host-passthrough_{context}"]
+= Enabling USB host passthrough
+
+You can enable USB host passthrough at the cluster level.
+
+You specify a resource name and USB device name for each device you want to add and then assign to a virtual machine (VM). You can allocate more than one device, each of which is known as a `selector` in the HyperConverged (HCO) custom resource (CR), to a single resource name. If you have multiple, identical USB devices on the cluster, you can choose to allocate a VM to a specific device.
+
+.Prerequisites
+
+* Enabling the passthrough requires cluster administrator privileges.
+
+.Procedure
+
+. Identify the USB device vendor and product by running the following command:
++
+[source,terminal]
+----
+$ lsusb
+----
+
+. Open the HCO CR by running the following commmand:
++
+[source,terminal]
+----
+$ oc edit hyperconverged kubevirt-hyperconverged -n openshift-cnv
+----
+
+. Edit the CR by performing the following actions, as shown in the example that follows:
+
+.. Enable the `HostDevices` feature gate by adding it to the `spec.featureGates` stanza.
+.. Add a USB device to the `permittedHostDevices` stanza.
+
++
+.Example YAML snippet
+[source,yaml]
+----
+apiVersion: hco.kubevirt.io/v1beta1
+kind: HyperConverged
+metadata:
+   name: kubevirt-hyperconverged
+   namespace: {CNVNamespace}
+spec:
+  featureGates:
+    hostDevices: True <1>
+# ...
+  configuration:
+    permittedHostDevices: <2>
+      usb: <3>
+        - resourceName: kubevirt.io/peripherals <4>
+          selectors:
+            - vendor: "045e"
+              product: "07a5"
+            - vendor: "062a"
+              product: "4102"
+            - vendor: "072f"
+              product: "b100"
```

Comment:
@tiraboschi Do we have this?

Reply:
No, actually only `PciHostDevices` and `MediatedDevices`.
see: https://github.com/kubevirt/hyperconverged-cluster-operator/blob/84c79b00720e32c54f1a25baff18a1e03f095586/api/v1beta1/hyperconverged_types.go#L505-L514

Reply:
@acardace,  is **permittedHostDevices** now available?

Reply:
yes, but the API actually looks different:

```
apiVersion: hco.kubevirt.io/v1beta1
kind: HyperConverged
metadata:
   name: kubevirt-hyperconverged
   namespace: {CNVNamespace}
spec:
   permittedHostDevices: (1)
     usb: (2)
       - resourceName: kubevirt.io/peripherals (3)
         selectors:
           - vendor: "045e"
             product: "07a5"
           - vendor: "062a"
             product: "4102"
          - vendor: "072f"
            product: "b100"
```

@tiraboschi can you double check?